core/domain/role_services.py

```diff
@@ -136,18 +143,23 @@
 #   this dict.
 ROLE_ACTIONS = {
     feconf.ROLE_ID_ADMIN: [
+        ACTION_CHANGE_QUESTION_STATUS,
         ACTION_CREATE_NEW_SKILL,
         ACTION_CREATE_NEW_TOPIC,
+        ACTION_CREATE_NEW_QUESTION,
         ACTION_DELETE_ANY_ACTIVITY,
         ACTION_DELETE_ANY_SKILL,
+        ACTION_DELETE_ANY_QUESTION,
```

Comment:
Here and others: probably not needed here if already in topic manager role. Permissions cascade.